stanford/roles/cluster/tasks/instance.yml

```diff
@@ -19,6 +19,7 @@
     volumes:
       - device_name: /dev/sda1
         volume_size: "{{ CLUSTER_EBS_SIZE }}"
```

Comment:
How about upping the size for edxapp cluster in light of what happened yesterday.

Reply:
Yup, I am; that's what prompted this too.
That change is coming in an of yet unopened PR.

Reply:
https://github.com/Stanford-Online/configuration/pull/234